### PR TITLE
Update vcpkg-tool to 2022-03-08.

### PR DESCRIPTION
**Pull request by BillyONeal**

### CODE REVIEW

**Review by JackBoosY — APPROVED** [review]

**Comment by BillyONeal** [comment]
PR run is expected to fail because determining parent hashes switches to main first and tries to use the broken tool. CI run running: https://dev.azure.com/vcpkg/public/_build/results?buildId=68426&view=results

**Comment by BillyONeal** [comment]
Ahem I mean https://dev.azure.com/vcpkg/public/_build/results?buildId=68428&view=results

**Comment by BillyONeal** [comment]
This is still red but it's still better than before.

File: scripts/azure-pipelines/test-modified-ports.ps1 [hunk 1]
```diff
@@ -165,9 +165,7 @@ if (($BuildReason -eq 'PullRequest') -and -not $NoParentHashes)
     $parentHashesFile = Join-Path $WorkingRoot 'parent-hashes.json'
     $parentHashes = @("--parent-hashes=$parentHashesFile")
     & git revert -n -m 1 HEAD | Out-Null
-    & "./vcpkg$executableExtension" ci $Triplet --dry-run --exclude=$skipList @hostArgs @commonArgs --no-binarycaching "--output-hashes=$parentHashesFile" `
-        | ForEach-Object { if ($_ -match ' dependency information| determine pass') { Write-Host $_ } }
```
Comment (dg0yt) [inline scripts/azure-pipelines/test-modified-ports.ps1:169]: Did this hide error output? Do we need the full standard output?

Comment (BillyONeal) [reply scripts/azure-pipelines/test-modified-ports.ps1:169]: Yes it did. Triggered the msi issue and that resulted in lots of confusion

**Pull request merged**